products/BellHybrid/services/db/agents/QuotesAgent.cpp

```diff
@@ -51,6 +60,72 @@ namespace Quotes
         return nullptr;
     }
 
+    auto QuotesAgent::handleAddNewEntry(std::shared_ptr<db::Query> query) -> std::unique_ptr<db::QueryResult>
+    {
+        const auto request = std::dynamic_pointer_cast<Messages::AddNewEntry>(query);
+        if (request == nullptr) {
+            auto response = std::make_unique<Messages::AddQuoteResponse>(false, 0);
```

Comment:
I'm wondering whether it would be a good idea to add a constructor to `Messages::AddQuoteResponse` (also `Messages::EditEntryResponse` and `Messages::DeleteQuoteResponse`) that will set those fields to default values needed when handling null request case.

IMO current solution is not very clear, as we're passing `false` and `0`, which are both not really meaningful. After adding new constructor such response would be created with no arguments `std::make_unique<Messages::AddQuoteResponse>();`

But if you prefer the current solution feel free to resolve, I'm not convinced which one is better TBH :wink:

Reply:
Good idea! Done.